src/app/page.tsx

```diff
@@ -1,53 +1,19 @@
-import Image from "next/image"
 import { auth } from "./auth"
+import Main from "@/components/main-content/main"
 
 export default async function Home() {
     const session = await auth()
+    interface Session {
```

Comment:
create a type.ts file and put this interface there since we might be reusing the session type in other places